doc/api/esm.md

```diff
@@ -1892,8 +1892,10 @@ The resolver can throw the following errors:
 **READ_PACKAGE_SCOPE**(_url_)
 
 > 1. Let _scopeURL_ be _url_.
-> 1. While _scopeURL_ is not the file system root,
+> 1. Do-while _scopeURL_ is not the file system root,
 >    1. If _scopeURL_ ends in a _"node_modules"_ path segment, return **null**.
+>    1. If the _scopeURL_ item is at the file system root, then
+>       1. Set _scopeURL_ to the file system root.
```

Comment:
These two lines can be removed as they seem redundant.

Reply:
Removed as the logic behind them has also been removed in https://github.com/nodejs/node/pull/34595#discussion_r469658001.